ReplicationAcceptance/ReplicationAcceptance/ChangeTrackerDelegate.m

```diff
@@ -0,0 +1,29 @@
+//
+//  ChangeTrackerDelegate.m
+//  ReplicationAcceptance
+//
+//  Created by Adam Cox on 1/8/15.
+//
+//
+
+#import "ChangeTrackerDelegate.h"
+#import "TDURLConnectionChangeTracker.h"
+
+@implementation ChangeTrackerDelegate
+
```

Comment:
need to add an -init method that initializes the block properties to blocks that just return. otherwise, if those blocks are not defined and they are called below, there will be a crash.